molecule/scenario_resources/tests/test_default.rb

```diff
@@ -8,3 +8,34 @@
     it { should be_enabled }
   end
 end
+
+'pulpcore-webserver' do |webserver|
+  describe port(80) do
+    it { should be_listening }
+  end
+
+  describe port(443) do
+    it { should be_listening }
+  end
+
+  describe http('http://localhost/pulp/api/v3/status',
+                 ssl_verify: false) do
+      its('status') { should eq 301 }
+    end
+  end
+
+  describe http('http://localhost/pulp/api/v3/status',
+                 ssl_verify: false, max_redirects: 1) do
+      its('status') { should eq 200 }
+      its('body') { should match /database_connection/ }
+    end
+  end
+
+  describe http('https://localhost/pulp/api/v3/status',
+                 ssl_verify: false) do
+      its('status') { should eq 200 }
+      its('body') { should match /database_connection/ }
+    end
+  end
+
+end
```

Comment:
Nice! But I thought this was failing due to the container permissions issue on GHA/Ubuntu. I wrote a long ticket after it derailed other changes I was working on.

Reply:
These tests are not being run:
https://github.com/pulp/pulp_installer/pull/356/checks?check_run_id=906763516#step:9:1438

Reply:
Ahh, that is the reason why my machine turns into a heir dryer, once molecule containers are installed. Do you think, we can at least ratelimit the restart attempts of pulp_services?